Data/DataModel/Prerequisites.cs

```diff
@@ -12,11 +12,21 @@ public class Prerequisites
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int ID { get; set; }
 
+        /// <summary>
+        /// The puzzle ID that depends on others.
```

Comment:
'others' can be easy to confuse here - thoughts on "The puzzle ID for which prerequisites are being set"? Same for the rest of the file